metabrainz/static/css/theme/variables.less

```diff
@@ -307,7 +307,7 @@
 // Navs
 // -------------------------
 
-@nav-link-padding: 10px 15px;
+@nav-link-padding: 10px 7.5px;
```

Comment:
7.5 is making it a bit less. It leaves this awkward space in the end.
I feel 10px is just fine, enough space and makes it look decent.

Here's how it looks with 7.5 - 

![screenshot from 2018-01-09 23-45-51](https://user-images.githubusercontent.com/12889549/34736109-d4c49546-f597-11e7-8089-30b07018ce2c.png)

And here's with 10px - 

![screenshot from 2018-01-09 23-46-33](https://user-images.githubusercontent.com/12889549/34736137-e9e92496-f597-11e7-907a-36392e1044d6.png)